btrack/napari/widgets/_general.py

```diff
@@ -1,6 +1,40 @@
 from __future__ import annotations
 
-from qtpy import QtWidgets
+from pathlib import Path
+
+from qtpy import QtCore, QtGui, QtWidgets
+
+
+def create_logo_widgets() -> dict[str, QtWidgets.QWidget]:
+    """Creates the widgets for the title, logo and documentation"""
+
+    title = QtWidgets.QLabel("<h3>Bayesian Tracker</h3>")
+    title.setAlignment(QtCore.Qt.AlignHCenter)
+    widgets = {"title": title}
+
+    logo = QtWidgets.QLabel()
+    logo.setPixmap(
+        QtGui.QPixmap(
```

Comment:
is it possible to make this resizable so that the logo size changes with the main widget? Or if not make the logo a little bit smaller

Reply:
Yes it is, but it can then stretch the logo text which looks weird

Reply:
i.e. 
![image](https://github.com/quantumjot/btrack/assets/15052188/82c465f1-5c9c-4748-8555-6d25e43c9b35)

can see if it can be done

Reply:
I've had a look. Struggling for now, so will leave it.